src/app/locales/zh_CN.ts

```diff
@@ -0,0 +1,444 @@
+  },
+  'forms AbstractFormControl': {
+    action: '如果您使用 Angular Forms，则 `AbstractFormControl.parent` 的类型现在包含 null。ng update 将自动迁移此项。但是在一个不太可能的情况下，您的代码是针对 undefined 和 strict equality 来测试父对象的，因此需要将其改为 `=== null`，因为父对象现在显式地用 `null` 初始化，而不是保持 undefined。'
+  },
+  'platform-webworker': {
+    action: '在 v8 中已弃用了很少使用的 `@angular/platform-webworker` 和 `@angular/platform-webworker-dynamic`，并已将其删除。在 Web worker 中运行 Angular 的部分内容是一项实验，在常见的用例中效果不佳。Angular 仍然对 <a href="https://angular.io/guide/web-worker" target="_blank">Web Workers</a> 有很好的支持。'
+  },
+  'v11 slice pipe typing': {
+    action: '`slice` 管道现在为未定义的输入值返回 null，这与大多数管道的行为一致。'
+  },
+  'v11 keyvalue typing': {
+    action: '`keyvalue` 管道已修复，可以报告具有数字 key 的输入对象，结果类型将包含键的字符串表示形式。已经是这种情况，并且仅对代码进行了更新以反映这一点。如果他们依赖不正确的类型，请更新管道输出的使用者。请注意，这不会影响输入值为 `Map` 的用例，因此，如果需要保留 `number`，这是一种有效的方法。'
+  },
+  'v11 number pipe typing': {
+    action: '现在，数字管道(`decimal`， `percent`， `currency`等)明确说明了接受哪些类型。'
+  },
+  'v11 date pipe typing': {
+    action: '现在，`date` 管道明确说明了接受哪些类型。'
+  },
+  'v11 datetime rounding': {
+    action: '当将日期时间格式的字符串以包含毫秒部分的格式传递给 `DatePipe` 时，现在毫秒将取整，而不是取最接近的毫秒。大多数应用程序不会受到此更改的影响。如果这不是期望的行为，则在将字符串传递给 `DatePipe` 之前，考虑对其进行预处理以舍入毫秒部分。'
+  },
+  '11 async pipe typing': {
+    action: '`async` 管道不再要求为键入为 undefined 的输入返回 undefined。请注意，代码实际上在未定义的输入上返回 null。'
+  },
+  'v11 case pipe update': {
+    action: '`uppercase` 和 `lowercase` 管道不再让虚假的值通过。现在，它们将 `null` 和 `undefined` 都映射到 `null`，并在无效输入(`0`， `false`， `NaN`)上引发异常。这与其他 Angular 管道匹配。'
+  },
+  'v11 router NavigationExtras typing': {
+    action: '如果您将路由(Router)与 `NavigationExtras` 一起使用，则新的类型允许传入类型为 `NavigationExtras` 的变量，但它们将不允许对象文字(object literals)，因为它们可能仅指定已知属性。他们也将不接受那些与 `Pick` 中的属性没有共同属性的类型。如果您受到此更改的影响，请仅从 NavigationExtras 中指定实际在各个函数调用中使用的属性，或在对象或变量上使用类型声明：`as NavigationExtras`。'
+  },
+  'v11 TestBed.overrideProvider': {
+    action: '在测试中，如果在 TestBed 初始化之后调用 `TestBed.overrideProvider`，则不再应用提供程序替代。此行为与其他替代方法(例如 `TestBed.overrideDirective` 等)一致，但它们会抛出错误来表明这一点。该检查以前在 TestBed.overrideProvider 函数中丢失。如果看到此错误，则应在完成 TestBed 初始化之前移动 `TestBed.overrideProvider` 调用。'
+  },
+  'v11 router RouteReuseStrategy': {
+    action: '如果使用路由(Router)的 RouteReuseStrategy，参数顺序已更改。先前在评估子路由时调用 `RouteReuseStrategy#shouldReuseRoute` 时，它们将使用被交换的 `future` 和 `current` 参数来调用。如果您的 `RouteReuseStrategy` 仅依赖于将来或当前的快照状态，您可能需要更新 `future` 和 `current`、`ActivateRouteSnapshots` 的 `shouldReuseRoute` 实现。'
+  },
+  'v11 locale data readonly': {
+    action: '如果您使用区域设置数据数组(locale data arrays)，则此 API 现在将返回只读数组。如果您要对它们进行突变(例如，调用 `sort()`，`push()`，`splice()`等)，则您的代码将不再编译。如果您需要更改数组，现在应该获取副本(例如，通过调用 `slice()`)并更改副本。'
+  },
+  'v11 CollectionChangeRecord': {
+    action: '在更改检测中，已删除 `CollectionChangeRecord`，而改用 `IterableChangeRecord`。'
+  },
+  'v11 forms async validators': {
+    action: '如果在初始化时对 `FormControl`、`FormGroup` 或 `FormArray` 类实例上定义的异步验证器使用 Angular Forms，则在异步验证器完成后，以前不会发出状态更改事件。这已被更改，因此status事件被发送到 `statusChanges` observable 中。如果代码依赖于旧的行为，则可以忽略此附加的状态更改事件。'
```

Comment:
> If you use Angular Forms with async validators defined at initialization time on class instances of `FormControl`, `FormGroup` or `FormArray` ,
> the status change event was not previously emitted once async validator completed.
> This has been changed so that the status event is emitted into the `statusChanges` observable.
> If your code relies on the old behavior, you can filter/ignore this additional status change event.

My understanding is that the original description said that the status event after the completion of the asynchronous validator was added. If you use the original code, you can ignore it.

There should be no problem in translating "filter/ignore" to "忽略"

Reply:
filter意味着人们可以使用rxjs filter（）管道过滤事件? 我不确定这是否是Angular团队的意思。

Reply:
这描述确实耐人寻味……